src/neptune_lightgbm/impl/__init__.py

```diff
@@ -201,7 +205,7 @@ def create_booster_summary(
     log_importances: bool = True,
     max_num_features: int = 10,
     list_trees: list = None,
-    log_trees_as_dataframe: bool = True,
+    log_trees_as_dataframe: bool = False,
```

Comment:
This might be seen as backward incompatibility. What do you think about releasing it as `0.10.0`?